Client/Repositories/Implementations/AnswerRepository.cs

```diff
@@ -18,14 +19,26 @@ public AnswerRepository(IHttpService httpService)
             this._httpService = httpService;
         }
 
-        public async Task<List<AnswerResponseDTO>> GetAnswersAsync(int idQuestion)
+        public async Task<List<AnswerResponseDTO>> GetAnswersAsync(int idQuestion, AnswersPageDTO productParams)
         {
-            return (await _httpService.Get<List<AnswerResponseDTO>>($"{url}/{idQuestion}")).Response;
+            var queryStringParam = new Dictionary<string, string>
+            {
+                ["pageSize"] = productParams.PageSize.ToString(),
```

Comment:
zamiast przesyłania zmiennej w DTO, proponuję dodać element pageSize do pliku Shared/Config.cs po czym wstrzyknąć go do repozytorium

Reply:
W przypadku dodania odpowiedzi przeładowane są wszystkie odpowiedzi pobrane z bazy wraz z tą nową. W tym przypadku wykorzystuje to pageSize (równe ilości pobranych aktualnie pobranych odpowiedzi) inne niż standardowe. Plus myślałem że ewentualnie w przyszłości w zależności od wymiarów ekranu różną ilość odpowiedzi jednorazowo ładować. W każdym bądź razie popatrzę i może dla przypadku ładowania w formie lazy loadingu będę ładował pageSize z Config.cs